client/components/domains/choose-domain-later/index.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import { useExperiment } from 'calypso/lib/explat';
+import ReskinSideExplainer from '../reskin-side-explainer';
+
+type Props = {
+	step: {
+		domainForm: {
+			loadingResults: boolean;
+			searchResults?: Array< any > | null;
+		};
+	};
+	handleDomainExplainerClick: () => void;
+	flowName: string;
+};
+
+export default function ChooseDomainLater( props: Props ) {
+	const [ isEscapeHatchShownOnce, setisEscapeHatchShownOnce ] = useState( false );
+	const [ isLoading, experimentAssignment ] = useExperiment(
```

Comment:
Normally I'd say we can consider to move this out and make them pure props since the parent component loads the same experiment anyway. However, doing this here enables a very interesting advantage of confirming the whole logic works with the ExPlat API by mocking `useExperiment()`. So this looks like an creative intentional design to me :)